### PR TITLE
Update commit

### PR DESCRIPTION
**Pull request by Svenum**
Update commit to fix EA-APP integration

### CODE REVIEW

**Comment by Svenum** [comment]
I had this build failure too on my machine, but after updating curl on my local machine this worked.

File: net.lutris.Lutris.yml [hunk 1]
```diff
@@ -571,10 +571,10 @@ modules:
     sources: *sdl12_compat_sources
 
   # Needed for minecraft
-  - name: java11
+  - name: java17
```
Comment (A6GibKm) [inline net.lutris.Lutris.yml:574]: Did you check that, indeed, you can run minecraft with this version?

Comment (Svenum) [reply net.lutris.Lutris.yml:574]: But I try it
Normally you need 4 Java versions to run all Minecraft versions.
I think it was so:
|Minecraft Version|Java Version|
|-|- |
|<1.17|8|
|<1.20|17|
|>1.20|21|

Comment (Svenum) [reply net.lutris.Lutris.yml:574]: Newes version works great, so

**Comment by strycore** [comment]
We're figuring out the gsm error here: https://github.com/flathub/net.lutris.Lutris/pull/428

I might as well bump the version to the latest commit as part of this PR too. 
We're not that far away from the next release so we're getting close to having a more recent build in Flathub  non-beta

**Comment by strycore** [comment]
For Minecraft, I would really appreciate if that could be part of a separate Flatpak but how to achieve this has never been clear and if I understand correctly isn't documented anywhere.

**Comment by Svenum** [comment]
> For Minecraft, I would really appreciate if that could be part of a separate Flatpak but how to achieve this has never been clear and if I understand correctly isn't documented anywhere.

You mean as separate flatpak as optional dependency, something like the mongohud flatpak?

**Comment by strycore** [comment]
Yes. Ideally, the Lutris flatpak should contain Gtk and some Python dependencies. Nothing else. 
Components needed to run games should be in their own Flatpak. We're never going to jam all dependencies needed by all games in a single Flatpak.

Sady, there's a lot of pushback against doing that sort of thing, even though I find this type of architecture much cleaner and is a really nice way to leverage Flatpak. 

I'm not even mentionning Snap where this sort of architecture is downright impossible.

**Comment by Svenum** [comment]
Makes more sense for me too. Its stupid to build a large flatpak just for features that you might not need.

**Comment by strycore** [comment]
I give up on the Flatpak. If anyone needs admin rights to anything to be able to maintain the Flatpak, I'll gladly provide it.

**Comment by strycore** [comment]
Closing this PR since we are now on a later commit in the beta branch.

Please submit another PR if we need other changes (such as Java updates)

**Pull request closed without merging**